components/com_users/tmpl/registration/default.php

```diff
@@ -34,10 +35,19 @@
 					<?php if (isset($fieldset->label)) : ?>
 						<legend><?php echo Text::_($fieldset->label); ?></legend>
 					<?php endif; ?>
-					<?php echo $this->form->renderFieldset($fieldset->name); ?>
+					<?php foreach ($fields as $field): ?>
+						<?php if ($field->name=="jform[captcha]"): ?>
+							<?php $captcha=$field; ?>
```

Comment:
```suggestion
						<?php if ($field->name === 'jform[captcha]'): ?>
							<?php $captcha = $field; ?>
```
Please use type safe comparison and correct code style (one space before and after assignment or comparison operators, single quotes for string)